build.sbt

```diff
@@ -27,10 +28,24 @@ lazy val builderArgument = settingKey[String](
 lazy val releaseTask = taskKey[Unit](
   "Publishes an artifact and optionally makes a release if version is not a snapshot")
 
+def latestRelease: String = {
+  val src = scala.io.Source.fromURL("https://api.github.com/repos/evansiroky/timezone-boundary-builder/releases/latest")
+  val json = src.mkString
+  json
+}
+
+def parseReleaseName(jsonString: String): String = {
+  val json:Option[Any] = JSON.parseFull(jsonString)
+  val map:Map[String,Any] = json.get.asInstanceOf[Map[String, Any]]
+  val name:String = map("name").asInstanceOf[String]
+  println("Current timezone release is : " + name)
+  name
+}
+
 lazy val core = (project in file("core"))
   .settings(commonSettings)
   .settings(
-    builderArgument := dataVersion,
+    builderArgument := parseReleaseName(latestRelease),
```

Comment:
This change makes it impossible to produce a build with non-latest source data version. It also introduces a possibility for Timeshape jar version and data version it contains to diverge. While probability of this happening is low, I'd rather like to avoid it altogether.

I propose not changing this line, but instead to introduce a separate check for latest source data version. This check will happen during resource generation (see code block starting with `log.info("Timeshape resource doesn't exist, creating")` below). If current value of `dataVersion` is older than latest one from Github API, log a warning message, but let the build continue with not the most up-to-date data version. What do you think?

Reply:
I was unsure of what would be the best approach for release matching and numbering between timeshape and  timezone-boundary-builder.

I noticed there is a issue for changing the timeshape release numbering to be more Java like so the exact string could not matter in the future.

I'll update the code with proposed changes and update the pull request.

Reply:
Following the discussion, I would ask you if could be possible a automatic magical release to the maven repository synchonized with timezone data release? Is this viable?

Reply:
Do you mean something like listening for new releases in https://github.com/evansiroky/timezone-boundary-builder/releases, and automatically send a PR to Timeshape upgrading to latest version? It looks somewhat similar to https://github.com/fthomas/scala-steward, although I'm not deeply familiar with how these tools work, and how viable it is to use the same approach here. But it might be a good idea to at least open an issue about this.

Reply:
Yes, somenthing like that, but I don't know if it is a good idea.
The scala-steward tools appears to be similar, but it needs a configuration in your project plus a trigger in any CI/computer (If I understood correctly).